code/modules/admin/view_variables/view_variables.dm

```diff
@@ -14,26 +14,30 @@
 	var/datum/asset/asset_cache_datum = get_asset_datum(/datum/asset/simple/vv)
 	asset_cache_datum.send(usr)
 
+	var/isappearance = isappearance(thing)
 	var/islist = islist(thing) || (!isdatum(thing) && hascall(thing, "Cut")) // Some special lists dont count as lists, but can be detected by if they have list procs
-	if(!islist && !isdatum(thing))
+	if(!islist && !isdatum(thing) && !isappearance(thing))
```

Comment:
```suggestion
	if(!islist && !isdatum(thing) && !isappearance)
```

no need to do the work twice if we're already cacheing it

Reply:
ye I forgot it lol